sherpa/astro/tests/test_astro_data_swift_unit.py

```diff
@@ -411,8 +411,8 @@ def test_can_use_swift_data(make_data_path, clean_astro_ui):
     # bin.
     #
     # Unfortunately, using a range of 0.3-8.0 gives 771 bins
-    # in XSPEC - channels 30 to 800 - but 772 bins in Sherpa,
-    # channels 30 to 801.
+    # in XSPEC - channels 30 to 800 - but a 770 bins in Sherpa,
```

Comment:
```suggestion
    # in XSPEC - channels 30 to 800 - but 770 bins in Sherpa,
```

Reply:
I'm going to hold off fixing this until I have a bunch of changes.

Reply:
Addressed in https://github.com/sherpa/sherpa/pull/1216/commits/73ca7cc38e045b2883205f5a11b497a90e8a49bf